include/asio/ssl/dtls/acceptor.hpp

```diff
@@ -767,6 +768,7 @@ class acceptor
       , ah_(std::move(ah))
```

Comment:
I think you want to be using `std::forward` in a deduced context, example:
https://github.com/boostorg/beast/blob/1da229a27c6f0539d422bcedbcf47cfe2517164a/include/boost/beast/http/impl/read.ipp#L66

You really should study Beast and the composed operation tutorial! This is a very complex topic and it is **super easy** to make mistakes (happens to me all the time).